src/main/java/se/simonsoft/cms/item/export/CmsExportJobBase.java

```diff
@@ -28,11 +28,17 @@
 
 	public CmsExportJobBase(CmsExportPrefix jobPrefix, String jobName, String jobExtension) {
 		
-		if (jobName == null || jobName.isEmpty()) {
-			throw new IllegalArgumentException("Not a valid export job name: " + jobName);
+		if (jobName == null) {
+			throw new IllegalArgumentException("Job name can not be null: " + jobName);
 		}
 		
-	    // TODO: Regex validating the export name. No slashes allowed etc.
+		if (jobName.startsWith("/")) {
+			throw new IllegalArgumentException("Job name can not be prefixed with slash: " + jobName);
+		}
+		
+		if (jobName.isEmpty()) {
+			throw new IllegalArgumentException("Job name can not be empty: " + jobName);
```

Comment:
What is the point of including jobName in error message when it is empty?

Makes more sense to have isEmpty test before startsWith test.

Reply:
Fixed.